client/packages/requisitions/src/RnRForms/DetailView/RnRFormLine.tsx

```diff
@@ -159,18 +166,16 @@ export const RnRFormLine = ({
 
       {/* Readonly calculated value */}
       <RnRNumberCell
-        readOnly
+        backgroundColor={readOnlyBackgroundColor}
         textColor={textColor}
         value={line.adjustedQuantityConsumed}
-        onChange={() => {}}
```

Comment:
Sorry, this was a bit of munging around - changed so if no `onChange`, must be readOnly, rather than sep prop. And just passing in the background colour rather than calling useTheme and re-calculating for every number input